Detectors/GlobalTrackingWorkflow/src/secondary-vertexing-workflow.cxx

```diff
@@ -79,13 +79,18 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   auto requireCTPLumi = configcontext.options().get<bool>("require-ctp-lumi");
 
   GID::mask_t src = allowedSources & GID::getSourcesMask(configcontext.options().get<std::string>("vertexing-sources"));
+
+  // don't loop over TPC tracks if not needed, but load them anyway for dEdx evaluation
+  bool excludeTPCtracks = !(src & GID::getSourceMask(GID::TPC)).any();
+  src = src | GID::getSourceMask(GID::TPC);
```

Comment:
Unconditional request of the TPC input will lead to a crash if for some reason the TPC is not in the run. 
I think it is better to make the `excludeTPCtracks` a data member of the SVertexerParams while keeping the TPC DetID request as it is now. Then, if the TPC is requested, it will be used for your trick and  `excludeTPCtracks` will steer the combinatorics. 
Once we will have the TPC PID assignment propagated via global tracks, no change in the DetID src will be needed.

and in case it is f